code/obj/artifacts/artifact_objects/implanter.dm

```diff
@@ -0,0 +1,62 @@
+/obj/artifact/implanter
+	name = "artifact implanter"
+	associated_datum = /datum/artifact/implanter
+
+/datum/artifact/implanter
+	associated_object = /obj/artifact/implanter
+	type_name = "Implanter"
+	rarity_weight = 250
+	validtypes = list("eldritch", "ancient", "wizard")
+	validtriggers = list(/datum/artifact_trigger/force,/datum/artifact_trigger/electric,/datum/artifact_trigger/heat,
+	/datum/artifact_trigger/radiation,/datum/artifact_trigger/carbon_touch,/datum/artifact_trigger/silicon_touch,
+	/datum/artifact_trigger/cold)
+	fault_blacklist = list(ITEM_ONLY_FAULTS)
+	activ_text = "opens up, revealing a complex array of thin tubes!"
+	deact_text = "closes itself up."
+	react_xray = list(9, 70, 75, 11, "SEGMENTED")
+
+	var/obj/item/implant/artifact/imp = null
+	var/notReady = FALSE
```

Comment:
double negative makes this harder to read- would prefer `var/ready = TRUE` (plus inverting the rest of the uses of this var)